lib/web_ui/lib/src/engine/canvaskit/initialization.dart

```diff
@@ -15,11 +15,15 @@ import 'canvaskit_api.dart';
 import 'fonts.dart';
 
 /// Whether to use CanvasKit as the rendering backend.
-bool get useCanvasKit => FlutterConfiguration.flutterWebAutoDetect ? _detectRenderer() : FlutterConfiguration.useSkia;
+final bool useCanvasKit = FlutterConfiguration.flutterWebAutoDetect
+  ? _detectedRenderer
+  : FlutterConfiguration.useSkia;
```

Comment:
do we support changing `FlutterConfiguration.flutterWebAutoDetect` dynamically? It seems weird that we wouldn't make the whole `useCanvasKit` property a `final` field set once on startup, not just `detectRenderer`.

Reply:
Not sure, but I don't think changing this at runtime would work - your app would be full of objects from the other renderer type